src/blockchain/Blockchain.ts

```diff
@@ -123,7 +123,8 @@ export default class Blockchain implements IBlockchainApi {
     })
   }
 
-  public async getNonce(accountAddress: string): Promise<u64> {
-    return this.api.query.system.accountNonce<u64>(accountAddress)
+  public async getNonce(accountAddress: string): Promise<Index> {
+    const x = (await this.api.query.system.account(accountAddress)).nonce
+    return x
```

Comment:
either return directly or use deconstruction.

```suggestion
    const { nonce } = await this.api.query.system.account(accountAddress)
    return nonce
```

Reply:
Oops. This is a result of me testing some stuff 🙈

Reply:
Fixed in 4e9832e